src/themes/shared/recipes/badge.ts

```diff
@@ -1,30 +1,29 @@
-import { ComponentStyleConfig, defineStyle } from '@chakra-ui/react';
+import { defineRecipe } from '@chakra-ui/react';
```

Comment:
This is a helper function to define the component theme. Check out [the docs](https://www.chakra-ui.com/docs/theming/recipes) for more info